src/main/java/uk/gov/hmcts/cmc/claimstore/models/ccj/PaymentSchedule.java

```diff
@@ -1,7 +1,17 @@
 package uk.gov.hmcts.cmc.claimstore.models.ccj;
 
 public enum PaymentSchedule {
-    EACH_WEEK,
-    EVERY_TWO_WEEKS,
-    EVERY_MONTH
+    EACH_WEEK("each week"),
+    EVERY_TWO_WEEKS("every two weeks"),
+    EVERY_MONTH("every month");
+
+    private String description;
+
+    PaymentSchedule(String description) {
```

Comment:
I tend to use private modifier for enumerated types the same as for utility classes.

Reply:
sounds like you are doing it wrong then:
![image](https://user-images.githubusercontent.com/21194782/31542787-592e24be-b00b-11e7-8b2f-8e3c2d537986.png)

Reply:
Right, although redundant it not always wrong :)